### PR TITLE
Fix peer deps

### PR DESCRIPTION
**Pull request by ditkachuk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+[Example](https://experium.github.io/react-designer/index.html).
```
Comment (Panacee) [inline README.md]: without `index.html`

**Comment by Panacee** [comment]
rebase

**Comment by ditkachuk** [comment]
rebased

File: example/package.json [hunk 2]
```diff
@@ -7,11 +7,12 @@
     "@experium/react-designer": "^2.0.0",
     "react": "^16.7.0",
     "react-dom": "^16.7.0",
+    "styled-components": "^4.1.3",
     "react-scripts": "2.1.3"
   },
   "scripts": {
     "start": "react-scripts start",
-    "build": "react-scripts build",
+    "build": "PUBLIC_URL=react-designer react-scripts build",
```
Comment (Panacee) [inline example/package.json]: For what?

Comment (ditkachuk) [reply example/package.json]: removed

**Pull request merged**